text.go

```diff
@@ -542,7 +543,7 @@ func (t *Text) ToPaths() ([]*Path, []color.RGBA) {
 			p := deco.ff.Decorate(deco.x1 - deco.x0)
 			p = p.Translate(deco.x0, line.y)
 			paths = append(paths, p)
-			colors = append(colors, deco.ff.color)
```

Comment:
For consistency we should probably rename this to `deco.face.Color` or `deco.Face.Color`.

Reply:
renamed!